bin/rl/rl.cpp

```diff
@@ -4167,8 +4170,8 @@ ProcessConfig
 
       // Walk the condition nodes looking for applicability.
 
-      ConditionNodeList * conditionNodeList = NULL;
-      ConditionNodeList * conditionNodeLast = NULL;
+      conditionNodeList = NULL;
+      conditionNodeLast = NULL;
```

Comment:
The following is valid, which means it is allowed under /permissive-:

    SomeType* ptr; //declared without an initializer, can skip over with a goto
    initializeSomeType(ptr);